tests/cgal/cgal_remesh_surface.cc

```diff
@@ -65,9 +66,18 @@ test()
   dealii_tria_to_cgal_surface_mesh(tria0, surface_mesh0);
   dealii_tria_to_cgal_surface_mesh(tria1, surface_mesh1);
 
-  // close the surfaces
-  CGAL::Polygon_mesh_processing::stitch_borders(surface_mesh0);
-  CGAL::Polygon_mesh_processing::stitch_borders(surface_mesh1);
+  Assert(surface_mesh0.is_valid() && surface_mesh1.is_valid(),
+         ExcMessage("The CGAL surface mesh is not valid."));
+  if (dim == 3)
+    {
+      Assert(CGAL::is_closed(surface_mesh0) && CGAL::is_closed(surface_mesh1),
+             dealii::ExcMessage("The CGAL mesh is not closed"));
```

Comment:
```suggestion
             ExcMessage("The CGAL mesh is not closed"));
```
for this and all other assertions, we don't need to specify the dealii namespace